inc/Engine/Optimization/Minify/JS/Combine.php

```diff
@@ -729,6 +729,7 @@ protected function get_excluded_external_file_path() {
 			'static.zdassets.com',
 			'feedbackcompany.com/widgets/feedback-company-widget.min.js',
 			'widget.gleamjs.io',
+			'beta.phonewagon.com',
```

Comment:
is there a possibility that the domain changes at some point? The `beta` subdomain raises doubt, maybe excluding the top domain instead would be safer?